docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       POSTGRES_INITDB_ARGS: "--auth='ident' --auth='trust'"
 
   rabbitmq:
-    image: rabbitmq:3.7-management
+    image: rabbitmq:3.8.20-management
```

Comment:
> RabbitMQ version 3.8.20 was the latest version Cachito wanted to work with. Cachito refused to run when I tried upgrading RabbitMQ to 3.9.X. What's the reason for this? According to https://www.rabbitmq.com/changelog.html, it appears we're using an old Earlang version for Cachito, is that correct? I ask because of Chen's comment.

@dcho5 Team RabbitMQ produces a [package](https://github.com/rabbitmq/erlang-rpm) stripped down to only provide those components needed to run RabbitMQ. This is the recommended option for Fedora OS [https://www.rabbitmq.com/install-rpm.html].
@tkdchen @brunoapimentel @MartinBasti  WDYT?